appserver/ha/ha-file-store/src/main/java/org/glassfish/ha/store/adapter/file/FileBackingStore.java

```diff
@@ -326,33 +326,16 @@ private boolean removeFile(final File file) {
         return success;
     }
 
-    private byte[] getSerializedState(K key, V value)
+    private byte[] getSerializedState(V value)
             throws BackingStoreException {
-
-        byte[] data = null;
-        ByteArrayOutputStream bos = new ByteArrayOutputStream();
-        ObjectOutputStream oos = null;
-        try {
-            oos = new ObjectOutputStream(bos);
+        try (ByteArrayOutputStream bos = new ByteArrayOutputStream();
+                ObjectOutputStream oos = new ObjectOutputStream(bos)) {
             oos.writeObject(value);
             oos.flush();
-            data = bos.toByteArray();
+            return bos.toByteArray();
```

Comment:
static analysis now should be able to work out that result is never null. Also removed the key that wasn't used and that made this somewhat confusing. The caller for this is the like change above.